crates/op-test-vectors/src/derivation.rs

```diff
@@ -1,8 +1,112 @@
 //! Module containing the derivation test fixture.
 
 use serde::{Deserialize, Serialize};
+use alloy::primitives::B256;
+use alloy::consensus::Blob;
+use anvil_core::eth::transaction::TypedTransaction;
 
 /// The derivation fixture is the top-level object that contains
 /// everything needed to run a derivation test.
-#[derive(Serialize, Deserialize, Debug, Default)]
-pub struct DerivationFixture {}
+#[derive(Serialize, Deserialize, Debug, Default, Eq, PartialEq)]
+pub struct DerivationFixture {
+    /// A list of chain transitions.
+    pub transitions: Vec<Transition>,
+    /// A list of L1 Blocks to derive from.
+    pub blocks: Vec<FixtureBlock>,
+}
```

Comment:
Im less familiar with the derivation pipeline, is there a reason that the `DerivationFixture` needs both `transitions` and `blocks` given that `blocks` contains the block number and block hash as well? 

Could we have something like the following and infer `previous_block` from the order of blocks in the vec?

```rust
pub struct DerivationFixture {
    /// A list of L1 Blocks to derive from.
    pub blocks: Vec<FixtureBlock>,
}
```

Reply:
We went with this design because it seemed like a more declarative way of saying there's a re-org. But I think we actually do have enough with the existing block information to derive the transitions.